docusaurus/sidebars.js

```diff
@@ -742,11 +758,6 @@ module.exports = {
         "release_notes/v-1.2",
         "release_notes/v-1.1",
         "release_notes/v-1.0",
-        "release_notes/aug_2024",
-        "release_notes/july_2024",
-        "release_notes/june_2024",
-        "release_notes/may_2024",
-        "release_notes/april_2024",
```

Comment:
Noting deletions here match insertions in the other file, but I don't have more context into 'why'.